.github/workflows/release.yml

```
@@ -0,0 +1,41 @@
+jobs:
+  build-wheels:
+    name: upload pybamm-cookie wheel artifacts
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout pybamm-cookie
+      - uses: actions/checkout@v4
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.12
+
+      - name: Build package
+        run: pipx run build
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v4.3.6
+        with:
+          name: pybamm_cookie_wheel
+          path: ./dist/*.whl
+          if-no-files-found: error
+
+  publish-pypi:
+    name: Upload package to PyPI
+    needs: [build-wheels]
+    runs-on: ubuntu-latest
+    environment: release
+    permissions:
+      id-token: write
+
+    steps:
+      - name: Download all artifacts
+        uses: actions/download-artifact@v4
+        with:
+          path: artifacts
```

```suggestion
        with:
          path: artifacts
          merge-multiple: true
```

This change will cause the artifacts to get downloaded into the `artifacts/` folder, and not inside `artifacts/sdist_and_wheel/`, which should be better.